lib/light-service/action.rb

```diff
@@ -9,7 +9,7 @@ def self.extended(base_class)
     def self.included(base_class)
       msg = "including LightService::Action is deprecated. " \
             "Please use `extend LightService::Action` instead"
-      ActiveSupport::Deprecation.warn(msg)
+      warn(StructuredWarnings::DeprecatedMethodWarning, msg)
```

Comment:
Here and wherever we thrown a `warn`, if this gem will remain in, I'd like to have a `class LightService::Deprecation < StructuredWarnings::Base; end` to use insted of directly expose the gem's built-in.

Reply:
I'm completely for that course of action.